app/src/main/java/helium314/keyboard/latin/LatinIME.java

```diff
@@ -1144,7 +1154,7 @@ public void onExtractedCursorMovement(final int dx, final int dy) {
 
     @Override
     public void hideWindow() {
-        if (mSuggestionStripView != null)
+        if (mSuggestionStripView != null && !mSettings.getCurrent().mAutoShowToolbar)
```

Comment:
Could you add some comment to the code on why hiding the toolbar depends on the auto-show setting? When reading just the code here, and not the entire PR one could easily assume it's a mistake.

Reply:
Why did you remove it? I just would have liked a comment in the code to avoid confusion, but now I am confused with this change...